pages/blog/build-a-mega-menu-with-react-bootstrap.mdx

```diff
@@ -0,0 +1,292 @@
+
+Since both our [core product](https://github.com/grouparoo/grouparoo) and our [website](https://github.com/grouparoo/www.grouparoo.com) are open source here at Grouparoo, let’s dive in and take a look at how we built out a mega menu using our stack.
+
+## First, What’s a Mega Menu?
+
+This!
+
+<Image
+  centered
+  alt="Completed Navbar at grouparoo.com"
+  src="build-a-mega-menu-with-react-bootstrap/final-on-www.png"
+  width={845}
+  height={277}
+/>
+
+A mega menu is a way to have multiple columns (or more!) of items within your dropdown menu. Some key things to consider:
+
+1. Make sure you have enough items within your dropdown to warrant a mega menu
+2. Figure out how you want to arrange the items within your menu
+
+Recently, we’ve been adding a lot of content to our site. We wanted to link more pages in our navigation, but still keep the top level navigation bar minimal. So we went with a mega menu.
+
+## Let’s Build
+
+For this tutorial, we'll be building this copycat mega menu:
+
+<Image
+  centered
+  alt="Copycat Navbar we'll be making"
+  src="build-a-mega-menu-with-react-bootstrap/finished-tutorial.png"
+  width={845}
+  height={269}
+/>
+
+Coffee themed, because of course. I've created a [basic boilerplate](https://github.com/grouparoo/nextjs-react-bootstrap-boilerplate)
+project with [NextJS](https://nextjs.org/), [React Bootstrap](https://react-bootstrap.github.io/),
+[SASS](https://sass-lang.com/), and [Fort Awesome](https://github.com/FortAwesome)
+already hooked up for you to follow along.
+
+React Bootstrap, and Bootstrap in general are great. The built-in component, classes, and utilities save me a lot of time building responsive layouts, and they're accessible by default. They’re also super flexible. For those less comfortable with front end, Bootstrap's [documentation](http://www.getbootstrap.com) is full of copy-and-pasteable examples to get started making modern, accessible sites. And once you've been working with Bootstrap for a while, it becomes almost second nature to Frankenstein pieces together like we're about to do.
+
+This is our `Navbar` element starting out:
+
+<Image
+  centered
+  alt="Starting tutorial navbar"
+  src="build-a-mega-menu-with-react-bootstrap/before-tutorial.png"
+  width={845}
+  height={234}
+/>
+
+When developing our mega menu for Grouparoo, I started with this Figma mockup. From there, I looked at what React Bootstrap components to use. We did end up making a few design changes further into the process, but it all started with:
+
+<Image
+  centered
+  alt="Mockups on Figma for our original"
+  src="build-a-mega-menu-with-react-bootstrap/mockups-screenshot.png"
+  width={845}
+  height={251}
+/>
+
+We're going to build something similar where the "events" link currently lives on our template.
+
+Let's start by adding the following into `/components/navigation.js` in place of the current `Nav.Link` for events:
+
+```
```

Comment:
try ```tsx so the code gets styles!

Reply:
To clarify, you mean just putting "tsx" after the three backticks?  Or converting the project to tsx?